### ISSUE TITLE
apt-get update fails

### ISSUE THREAD

**Issue by nicholasmarais1158** [mingchen/docker-android-build-box#28]
Tag: 1.8.0

I run `apt-get update` as an early step in my build.

Recently, I'm getting the following error;
```
+ apt-get update
Ign:1 http://security.ubuntu.com/ubuntu artful-security InRelease
Err:2 http://security.ubuntu.com/ubuntu artful-security Release
  404  Not Found [IP: 91.189.91.26 80]
Get:3 https://deb.nodesource.com/node_10.x artful InRelease [4,611 B]
Ign:4 http://archive.ubuntu.com/ubuntu artful InRelease
Ign:5 http://archive.ubuntu.com/ubuntu artful-updates InRelease
Get:6 https://deb.nodesource.com/node_10.x artful/main amd64 Packages [767 B]
Ign:7 http://archive.ubuntu.com/ubuntu artful-backports InRelease
Err:8 http://archive.ubuntu.com/ubuntu artful Release
  404  Not Found [IP: 91.189.88.161 80]
Err:9 http://archive.ubuntu.com/ubuntu artful-updates Release
  404  Not Found [IP: 91.189.88.161 80]
Err:10 http://archive.ubuntu.com/ubuntu artful-backports Release
  404  Not Found [IP: 91.189.88.161 80]
Reading package lists...
E: The repository 'http://security.ubuntu.com/ubuntu artful-security Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu artful Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu artful-updates Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu artful-backports Release' does not have a Release file.
```

My last successful build was 5th May 2019, and only noticed this error today.

Could this be related to Ubuntu 17.10 being "End of Life"? When I browse http://archive.ubuntu.com/ubuntu/dists/ I don't see directories for "artful".

Would this simply do the job?
```
- FROM ubuntu:17.10
+ FROM ubuntu:18.04
```

Any foreseen implications of upgrading to 18.04?

**Comment by joshuabergeron** [comment]
I have run into this problem today as well @nicholasmarais1158 and to answer your question about 18.04, my attempts to build using ubuntu:18.04 as the base were not successful (yet).

**Comment by mingchen** [comment]
PR is welcome!

**Comment by joshuabergeron** [comment]
It should be incoming soon! :) Only minor changes were required.

**Comment by mingchen** [comment]
Thanks @joshuabergeron for PR. PR merged and have a new tag `1.9.0`.
Please use tag `1.9.0` or `latest` to solve this issue.

**Comment by nicholasmarais1158** [comment]
Thanks @joshuabergeron!

Tested and working for react-native (v0.59.5) builds.
